.storybook/preview-head.html

```diff
@@ -0,0 +1 @@
+<script src="/scripts/mtiFontTrackingCode.js"></script>
```

Comment:
What does this code do?

Reply:
We load fonts that are licensed to fast.fonts.net, and in order to officially use them, we need to load this script to tell fonts.net we use it

Reply:
Does that mean that, sites that use this reusable component lib, also have to load that script?

Reply:
Yes, we should document this and add all the other fonts (with CSS fontface declarations) to asc-assets. I didn't add these in this branch because I think it's out of scope for this ticket, but did thought it would be a good idea to add this script already as it's a small effort